lib/options.ts

```diff
@@ -97,7 +97,12 @@ It should be a PlainObject consists of four properties: `'left'`, `'right'`, `'u
 **Note:** Assigning an empty string to any of these properties makes SpatialNavigation go nowhere at that direction.
 
    */
-  leaveFor?: any;
+  leaveFor?: {
+    left?: string;
+    right?: string;
+    up?: string;
+    down?: string;
+  };
```

Comment:
It seems that it can also be an array of or a single DOM element.
I yeeted JQuery, not sure if i got all of it out.
https://github.com/luke-chang/js-spatial-navigation#leavefor

Reply:
Take a look at this commit. Also added a license comment on the definition file you made.
https://github.com/Syncronet-APS/zome-vjsn/commit/14e8157854ecf8f644ea1d122be50d349d36c684

Reply:
Shouldn't the `LICENCE` file be enough?

> If it is not possible or desirable to put the notice in a particular file, then You may include the notice in a location (such as a LICENSE file in a relevant directory) where a recipient would be likely to look for such a notice.

https://www.mozilla.org/en-US/MPL/2.0/